tests/functional/tests.py

```diff
@@ -447,25 +940,21 @@ def main():
     is_s3 = server_endpoint.startswith("s3.amazonaws")
     client = Minio(server_endpoint, access_key, secret_key, secure=secure)
     # Check if we are running in the mint environment.
-    is_mint_env = os.path.exists(os.getenv('DATA_DIR', '/mint/data'))
-
-    _http = urllib3.PoolManager(
-        cert_reqs='CERT_REQUIRED',
-        ca_certs=certifi.where()
-    )
-
-    # Get unique bucket_name, object_name.
-    bucket_name = uuid.uuid4().__str__()
-    object_name = uuid.uuid4().__str__()
+    data_dir = os.getenv('DATA_DIR')
+    if data_dir == None:
+       os.environ['DATA_DIR'] = data_dir = '/mint/data'
+    is_mint_env = (os.path.exists(data_dir) and
+                  os.path.join(data_dir, 'datafile-1-MB') and
```

Comment:
os.path.join(data_dir, 'datafile-1-MB') returns a string, so this is always True, did you want os.path.exists(os.path.join, ..) instead ?

Reply:
Yes. That was the my purpose. Thank you for catching it.